azure/templates/deploy-service.yml

```diff
@@ -182,16 +183,27 @@ steps:
       export APIGEE_ORGANIZATION="nhsd-${{ parameters.apigee_organization }}"
       export RELEASE_RELEASEID="$(Build.BuildId)"
       export PING="${{ parameters.ping }}"
-      if [[ -n "${{ parameters.spec_file }}" ]]; then
-        export SPEC_FILE="$(SERVICE_DIR)/${{ parameters.spec_file }}"
-      fi
-      export FRIENDLY_NAME="$(FRIENDLY_NAME)"
-      export VISIBLE="${{ parameters.make_spec_visible }}"
-      export REQUIRE_CALLBACK_URL="${{ parameters.portal_api_requires_callback_url }}"
 
       make --no-print-directory -C $(UTILS_DIR)/ansible deploy-apigee-proxy
-      if [[ ! -z "${PRODUCT_DISPLAY_NAME}" ]]; then
-          make --no-print-directory -C $(UTILS_DIR)/ansible deploy-apigee-product-and-spec
+
+      if [ -f $(SERVICE_DIR)/manifest.yml ] || [ -f $(SERVICE_DIR)/manifest_template.yml ]; then
```

Comment:
For next ticket: templating in the build pipeline so the deploy is only ever seeing fully resolved manifests